src/test/java/simba/ui/SimbaTest.java

```diff
@@ -0,0 +1,22 @@
+package simba.ui;  //same package as the class being tested
```

Comment:
```suggestion
// This test class belongs to the same package as the class being tested.
package simba.ui;
```

Suggestion:
Move the comment to a separate line and ensure there is a blank line after the package declaration.